### PR TITLE
Panel: onClosed and onOpened fixed in React 18 strict mode

### LINKED ISSUE #26256: [Bug]: onOpened is not triggered on Panel component in React 18
### Library

React / v8 (@fluentui/react)

### System Info

```shell
System:
    OS: Windows 10 10.0.22000
    CPU: (12) x64 Intel(R) Core(TM) i5-10500 CPU @ 3.10GHz
    Memory: 14.79 GB / 31.78 GB
  Browsers:
    Edge: Spartan (44.22000.120.0), Chromium (108.0.1462.46)
    Internet Explorer: 11.0.22000.120
```

### Are you reporting Accessibility issue?

None

### Reproduction

https://codesandbox.io/s/peaceful-cache-0vhotd?file=/src/App.js

## Actual Behavior
If a function is specified for the onOpened function of the Panel and the panel gets opened, nothing happens, the callback is not triggered.

## Expected Behavior
The callback function should get executed

---
This issue seems similar: #11782

### Logs

_No response_

### Requested priority

Blocking

### Products/sites affected

_No response_

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

### PR DESCRIPTION
**Pull request by micahgodbolt**
Fixes #26256

Better approach now. Moving async AND event to componentDidMount will make sure they are instantiated each time the component is mounted (which is twice in R18 strict)

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/cd66dc983d876cfe097e92598e1f002df106edfb'>cd66dc983d876cfe097e92598e1f002df106edfb</a></sub>
<div id="bundleSizeComment9423"/>

File: apps/react-18-tests-v8/src/App.tsx [hunk 1]
```diff
@@ -1,8 +1,10 @@
 import * as React from 'react';
```
Comment (fabricteam) [inline apps/react-18-tests-v8/src/App.tsx:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26749&commitId=9d299e9fbc6f53634ad3d39c586e999b461f9ba5&env=prod)

✅ There was 1 screenshots added, 0 screenshots removed, 1044 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Image.Fit- center, image larger.default.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/GroupedList-mount_base.html">2328</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/GroupedList-mount.html">2682</a></td>
            <td>2</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/GroupedList-mount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/BaseButton-mount_base.html">1507</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/BaseButton-mount.html">1562</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/Breadcrumb-mount_base.html">3663</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/Breadcrumb-mount.html">3869</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/Checkbox-mount_base.html">3371</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/Checkbox-mount.html">3335</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/CheckboxBase-mount_base.html">2992</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/CheckboxBase-mount.html">2989</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/ChoiceGroup-mount_base.html">5503</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/ChoiceGroup-mount.html">5634</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/ComboBox-mount_base.html">1547</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/ComboBox-mount.html">1600</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/CommandBar-mount_base.html">12160</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/CommandBar-mount.html">12071</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/ContextualMenu-mount_base.html">17525</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/ContextualMenu-mount.html">17242</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/DefaultButton-mount_base.html">1790</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/DefaultButton-mount.html">1774</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/DetailsRow-mount_base.html">4406</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/DetailsRow-mount.html">4455</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/DetailsRowFast-mount_base.html">4340</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/DetailsRowFast-mount.html">4508</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/DetailsRowNoStyles-mount_base.html">4237</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/DetailsRowNoStyles-mount.html">4290</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/Dialog-mount_base.html">3908</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/Dialog-mount.html">3856</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/DocumentCardTitle-mount_base.html">732</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/DocumentCardTitle-mount.html">724</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/Dropdown-mount_base.html">4027</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/Dropdown-mount.html">4076</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/FocusTrapZone-mount_base.html">2554</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/FocusTrapZone-mount.html">2545</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/FocusZone-mount_base.html">2394</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/FocusZone-mount.html">2429</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/GroupedList-mount_base.html">2328</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/GroupedList-mount.html">2682</a></td>
            <td>2</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/GroupedList-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/GroupedList-virtual-rerender_base.html">1373</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/GroupedList-virtual-rerender.html">1429</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">2122</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/GroupedList-virtual-rerender-with-unmount.html">2108</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/GroupedListV2-mount_base.html">706</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/GroupedListV2-mount.html">725</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/GroupedListV2-virtual-rerender_base.html">686</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/GroupedListV2-virtual-rerender.html">700</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">709</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">712</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/IconButton-mount_base.html">2436</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/IconButton-mount.html">2387</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/Label-mount_base.html">911</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/Label-mount.html">925</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/Layer-mount_base.html">5482</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/Layer-mount.html">5472</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/Link-mount_base.html">1035</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/Link-mount.html">1044</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/MenuButton-mount_base.html">2078</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/MenuButton-mount.html">2092</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/MessageBar-mount_base.html">2922</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/MessageBar-mount.html">2919</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/Nav-mount_base.html">4469</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/Nav-mount.html">4131</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/OverflowSet-mount_base.html">1702</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/OverflowSet-mount.html">1698</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/Panel-mount_base.html">3256</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/Panel-mount.html">3204</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/Persona-mount_base.html">1609</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/Persona-mount.html">1628</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/Pivot-mount_base.html">2071</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/Pivot-mount.html">2073</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/PrimaryButton-mount_base.html">1898</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/PrimaryButton-mount.html">1937</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/Rating-mount_base.html">8736</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/Rating-mount.html">8807</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/SearchBox-mount_base.html">1902</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/SearchBox-mount.html">1915</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/Shimmer-mount_base.html">3515</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/Shimmer-mount.html">3501</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/Slider-mount_base.html">2680</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/Slider-mount.html">2599</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/SpinButton-mount_base.html">6345</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/SpinButton-mount.html">5811</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/Spinner-mount_base.html">1066</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/Spinner-mount.html">984</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/SplitButton-mount_base.html">3917</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/SplitButton-mount.html">3894</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/Stack-mount_base.html">1038</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/Stack-mount.html">1016</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/StackWithIntrinsicChildren-mount_base.html">2920</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/StackWithIntrinsicChildren-mount.html">2877</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/StackWithTextChildren-mount_base.html">6012</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/StackWithTextChildren-mount.html">5922</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/SwatchColorPicker-mount_base.html">12778</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/SwatchColorPicker-mount.html">13022</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/TagPicker-mount_base.html">3402</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/TagPicker-mount.html">3294</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/TeachingBubble-mount_base.html">105907</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/TeachingBubble-mount.html">104893</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/Text-mount_base.html">1003</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/Text-mount.html">1002</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/TextField-mount_base.html">2008</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/TextField-mount.html">2063</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/ThemeProvider-mount_base.html">1871</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/ThemeProvider-mount.html">1947</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/ThemeProvider-virtual-rerender_base.html">1380</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/ThemeProvider-virtual-rerender.html">1344</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">2615</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">2697</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/Toggle-mount_base.html">1406</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/Toggle-mount.html">1394</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/buttonNative-mount_base.html">682</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26749/perf-test/buttonNative-mount.html">677</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

File: packages/react/src/components/Panel/Panel.base.tsx [hunk 2]
```diff
@@ -397,6 +398,7 @@ export class PanelBase extends React.Component<IPanelProps, IPanelState> impleme
   };
 
   private _updateFooterPosition(): void {
+    console.log('update');
```
Comment (sopranopillow) [inline packages/react/src/components/Panel/Panel.base.tsx:401]: should this be removed?
```suggestion
```

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

File: apps/react-18-tests-v8/src/components/Panel.Basic.Example.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
```
Comment (fabricteam) [inline apps/react-18-tests-v8/src/components/Panel.Basic.Example.tsx:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 4 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26749&commitId=3e6b16c96cc11c119a3d01773cf207fba35efe52&env=prod)

✅ There was 0 screenshots added, 4 screenshots removed, 1041 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv8"/>

File: change/@fluentui-react-eff13953-3080-4ee9-8ed5-6a778a7f692f.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Panel: Fix async and events to work in concurrent mode",
```
Comment (sopranopillow) [inline change/@fluentui-react-eff13953-3080-4ee9-8ed5-6a778a7f692f.json:3]: Based on v8's changelog I think this should be:

```suggestion
  "comment": "fix: Panel async and events to work in concurrent mode",
```

**Review by sopranopillow — APPROVED** [review]
aside from the minor comment, lgtm

**Review by khmakoto — APPROVED** [review]

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

**Pull request merged**